### ISSUE TITLE
Confusion regarding 10x technology used for each sample only having two fastq.gz?

### ISSUE THREAD

**Issue by qyzhu0627** [zhangzlab/covid_balf#19]
Hello all@zhangzlab ,
Thanks for  nice work and  public data! I want to reanalyse this data from fastq, but I could  only get two fastq.gz from SRR of each sample. Can you help me?
Thanks in advance!

**Comment by zhangzlab** [comment]
Yes, we have merged all fastq files of R1 and R2 into one file, so you get two fastq.gz for each sample.

**Comment by qyzhu0627** [comment]
@zhangzlab Thanks for reply！But  I encountered another question when I run
  "**cellranger count --id=C141 --localcores=30 --transcriptome=/data1/xinguan/refdata-gex-GRCh38-2020-A/fasta/GRCh38-2020-A_NC_045512 --fastqs=./ --sample=C141   --expect-cells=5000 --nosecondary**"

It reported Error"2022-05-18 15:20:17 [runtime] (failed)          ID.C141.SC_RNA_COUNTER_CS.SC_MULTI_CORE.MULTI_CHEMISTRY_DETECTOR._GEM_WELL_CHEMISTRY_DETECTOR.DETECT_COUNT_CHEMISTRY
**[error] Pipestance failed. Error log at:
C141/SC_RNA_COUNTER_CS/SC_MULTI_CORE/MULTI_CHEMISTRY_DETECTOR/_GEM_WELL_CHEMISTRY_DETECTOR/DETECT_COUNT_CHEMISTRY/fork0/chnk0-u237a849e31/_errors
Log message:
FASTQ header mismatch detected at line 4 of input files "/data1/xinguan/C141/C141_S1_L001_R1_001.fastq" and "/data1/xinguan/C141/C141_S1_L001_R2_001.fastq": file: "/data1/xinguan/C141/C141_S1_L001_R1_001.fastq", line: 4**

 **Deatail of R1 and R2 as follows:**
 **C141_S1_L001_R1_001.fastq** :
@SRR11181954.1 1 length=26
GAACATCGTAACTGTATATTATTCTC
+SRR11181954.1 1 length=26
DCCC@;8C@7936;<->440.):8->
@SRR11181954.2 2 length=26
AAGCCGCAGTGCGATGGTACACAGCG
+SRR11181954.2 2 length=26
B:DCDDB?CBC=<::BA6<>.@.9(A
@SRR11181954.3 3 length=26
ACTGCTCCAGCCTGTGCCTGATGGTG

**C141_S1_L001_R2_001.fastq**:
@SRR11181954.1412724794 1412724794 length=100
GTCTGGCCAGCTGGTGAACTGAATGTGAGTCACCTCTCTTCCAGTTGCTTTTTCTTTTTTATTTACAATGTTCAATTTCTGAATGATGTAAGCTGGACAT
+SRR11181954.1412724794 1412724794 length=100
BCBBB6B4BCBBBB;BBAA@CBCB:ACD@>9D95C5?BC@@A@B:CB?A6::B@?>?@@@B<;BA=BA7E3?ACB7AA@B>A@9?<CB+@C1AACC(:A>
@SRR11181954.1412724795 1412724795 length=100
ATTGCGCCAGGTTTCAATTTCTATCGCCTATACTTTATTTGGGTAAATGGTTTGGCTAAGGTTGTCTGGTAGTAAGGTGGAGTGGGTTTGGGGCTAGGCT
+SRR11181954.1412724795 1412724795 length=100
<A@DDCCDCCB1@A<AD@=BBCCCAC;DCADCCCCBCDBDDDC=CACCACC6CD@BDACEC:BEAC>DB@CEBACCDDDCEE=CDC@4>CDDC(DCCD%=
@SRR11181954.1412724796 1412724796 length=100
CCTTTCCTGTTCACTCTACCCTTTGACTCTAAATCTCAAAGCCAGTGTTGGGGCCCAGTGGCTCCATTCGATTGAAACATGGCCAATGATATCCAAGAGC

**Comment by zhangzlab** [comment]
If the R1 and R2 are not in the same order, you can repair the order using other tools such as repair.sh in BBMap.

**Comment by qyzhu0627** [comment]
@zhangzlab The R1 and R2 indeed mismatch in order and I followed your advice , running
"repair.sh -Xmx440g in1=C152_S1_L001_R1_001.fastq.gz in2=C152_S1_L001_R2_001.fastq.gz out1=C152_S1_L001_R1.fastq.gz out2=C152_S1_L001_R2.fastq.gz".
unfortunately, it took too long and failed with empty files,reporting
"java.lang.Exception:
Mismatch between length of bases and qualities for read 403176504 (id=SRR11537951.500221381 500221381 length=26).
qualities=11, # bases=26
ECEEDDEEEED
GATCGTAGTATCAGTCACCATATAGG"
This can be bypassed with the flag 'tossbrokenreads' or 'nullifybrokenquality'"
I'm expecting your reply!

**Comment by qyzhu0627** [comment]
@zhangzlab  I'm being very distressed to use  repair.sh in BBMap to  reorder R1 and R2 in the same  order. Can you be kind to let me know how to use repair.sh in this case? Many thanks !
